akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -574,14 +754,56 @@ returned `Future` with a value. The `ask` operation involves creating
 an internal actor for handling this reply, which needs to have a timeout after
 which it is destroyed in order not to leak resources; see more below.
 
+@@@
+
+@@@ div { .group-java }
+
+This example demonstrates `ask` together with the `pipeTo` pattern on
+futures, because this is likely to be a common combination. Please note that
+all of the above is completely non-blocking and asynchronous: `ask` produces
+a `Future`, two of which are composed into a new future using the
+`Futures.sequence` and `map` methods and then `pipe` installs
+an `onComplete`-handler on the future to effect the submission of the
+aggregated `Result` to another actor.
+
+Using `ask` will send a message to the receiving Actor as with `tell`, and
+the receiving actor must reply with `getSender().tell(reply, getSelf())` in order to
+complete the returned `Future` with a value. The `ask` operation
+involves creating an internal actor for handling this reply, which needs to
+have a timeout after which it is destroyed in order not to leak resources; see
+more below.
+
+@@@ 
```

Comment:
In a number of places (for example here) `div`s are used even when the paragraphs are pretty similar. It might be worth merging them together and using the `@java`/`@scala` directives for the parts that are different?